package.json

```diff
@@ -4,17 +4,26 @@
     "dev": "nuxi dev",
     "tw": "tailwind-config-viewer --open --port 3001",
     "build": "nuxi build",
-    "start": "node .output/server/index.mjs"
+    "start": "node .output/server/index.mjs",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore ."
   },
   "devDependencies": {
     "@nuxt/postcss8": "^1.1.3",
     "@tailwindcss/typography": "^0.5.2",
-    "autoprefixer": "^10.4.0",
+    "autoprefixer": "^10.4.2",
     "nuxt3": "latest",
     "postcss": "^8.4.5",
     "tailwind-config-viewer": "^1.6.2",
     "tailwindcss": "^3.0.6",
-    "vue-gtag-next": "^1.14.0"
+    "vue-gtag-next": "^1.14.0",
```

Comment:
Why do you want to collect google analitics?

Reply:
the dependency was already there.